### ISSUE TITLE
Configuration Issue

### ISSUE THREAD

**Issue by dmueller-sa** [KenticoDevTrev/XperienceCommunity.WidgetFilter#2]
Hello. I'm running into this issue in Program.cs > CreateHostBuilder() after adding the services.AddWidgetFilter() service registration to startup.cs and attempting to run the application. Does the WidgetFilter service have a required configuration for ApplicationUser that is different from that of the vanilla Kentico 13 .Net Core installation? This appears to be the case as I've not added any special configuration pertaining to Microsoft.AspNetCore.Identity.UserManager. Thank you!

Inner Exception 1:
InvalidOperationException: Error while validating the service descriptor 'ServiceType: XperienceCommunity.WidgetFilter.IWidgetPermissionFilter Lifetime: Scoped ImplementationType: XperienceCommunity.WidgetFilter.WidgetPermissionFilter': Unable to resolve service for type 'Microsoft.AspNetCore.Identity.UserManager`1[Kentico.Membership.ApplicationUser]' while attempting to activate 'XperienceCommunity.WidgetFilter.WidgetPermissionFilter'.

Inner Exception 2:
InvalidOperationException: Unable to resolve service for type 'Microsoft.AspNetCore.Identity.UserManager`1[Kentico.Membership.ApplicationUser]' while attempting to activate 'XperienceCommunity.WidgetFilter.WidgetPermissionFilter'.

**Comment by KenticoDevTrev** [comment]
Yes you need to set up authentication, as the filter has role based filtering.

You can see an example from our baseline below, look to the authentication section. 

https://github.com/HBSTech/Kentico13CoreBaseline/blob/master/MVC/MVC/App_Start/StartupConfig.cs

**Comment by dmueller-sa** [comment]
> Yes you need to set up authentication, as the filter has role based filtering.
> 
> You can see an example from our baseline below, look to the authentication section.
> 
> https://github.com/HBSTech/Kentico13CoreBaseline/blob/master/MVC/MVC/App_Start/StartupConfig.cs

Thanks for the pointer. I thought it was something like that. I'll go by the example and hopefully won't need to bother you on this anymore. I appreciate your responsiveness and direction. -Dave
